configure.py

```diff
@@ -1097,6 +1098,9 @@ def configure_node(o):
   else:
     o['variables']['node_target_type'] = 'executable'
 
+def configure_napi(output):
+  output['variables']['napi_build_version'] = getnapibuildversion.get_napi_version()
```

Comment:
Long line. Can you wrap this at 80 columns or write it like this?
```python
version = getnapibuildversion.get_napi_version()
output['variables']['napi_build_version'] = version
```

Reply:
Thanks for review. I modified the code like you suggested.